rasa/server.py

```diff
@@ -377,6 +378,28 @@ def validate_request_body(request: Request, error_message: Text) -> None:
         raise ErrorResponse(HTTPStatus.BAD_REQUEST, "BadRequest", error_message)
 
 
+def validate_events_in_request_body(request: Request) -> None:
+    """Validates events format in request body."""
+    from jsonschema import validate
+    from jsonschema import ValidationError
+
+    if not isinstance(request.json, list):
+        events = [request.json]
+    else:
+        events = request.json
+
+    try:
+        validate(events, EVENTS_SCHEMA)
+    except ValidationError as error:
+        raise ErrorResponse(
+            HTTPStatus.BAD_REQUEST,
+            "BadRequest",
+            f"Failed to validate the events format. "
+            f"For more information about the format visit: "
+            f"https://rasa.com/docs/action-server/pages/action-server-api. Error: {error}",
```

Comment:
can we use our docs constants for this?

Reply:
I just found the base URL constant applicable and used this - let me know if that doesn't look ok!